ovn-tester/ovn_workload.py

```diff
@@ -462,65 +535,70 @@ def default_deny(self):
             'allow')
 
     @ovn_stats.timeit
-    def allow_within_namespace(self):
+    def allow_within_namespace(self, family=4):
         self.enforce()
+
+        addr_set = f'self.addr_set{family}.name'
         self.nbctl.acl_add(
             self.pg.name, 'to-lport', ACL_NETPOL_ALLOW_PRIO, 'port-group',
-            f'ip4.src == \\${self.addr_set.name} && '
-            f'outport == @{self.pg.name}',
+            f'ip4.src == \\${addr_set} && outport == @{self.pg.name}',
             'allow-related'
         )
         self.nbctl.acl_add(
             self.pg.name, 'to-lport', ACL_NETPOL_ALLOW_PRIO, 'port-group',
-            f'ip4.dst == \\${self.addr_set.name} && '
-            f'inport == @{self.pg.name}',
+            f'ip4.dst == \\${addr_set} && inport == @{self.pg.name}',
             'allow-related'
         )
 
     @ovn_stats.timeit
-    def allow_cross_namespace(self, ns):
+    def allow_cross_namespace(self, ns, family=4):
```

Comment:
I think we don't need the default `family=4`.  Let's just pass it explicitly.

Reply:
@dceara ack, I will fix it